### ISSUE TITLE
Bias Nodes, Epochs, and Multiple Hidden Layers You can experiment by changing:

### ISSUE THREAD

**Issue by triptithawait** [triptithawait/skills-copilot-codespaces-vscode#7]
use_bias=True or False in build_model()
epochs=10, 20, etc. in model.fit()
hidden_layers=[256, 128, 64] for deeper networks

**Comment by triptithawait** [comment]
from tensorflow.keras.datasets import mnist
from tensorflow.keras.utils import to_categorical
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Dense

# Load and preprocess MNIST data
(X_train, y_train), (X_test, y_test) = mnist.load_data()
X_train = X_train.reshape(-1, 28*28).astype('float32') / 255
X_test = X_test.reshape(-1, 28*28).astype('float32') / 255
y_train = to_categorical(y_train, 10)
y_test = to_categorical(y_test, 10)

# Define model builder
def build_model(hidden_layers=[256, 128, 64], use_bias=True):
    model = Sequential()
    model.add(Dense(hidden_layers[0], input_shape=(784,), activation='relu', use_bias=use_bias))
    for units in hidden_layers[1:]:
        model.add(Dense(units, activation='relu', use_bias=use_bias))
    model.add(Dense(10, activation='softmax', use_bias=use_bias))
    model.compile(optimizer='adam', loss='categorical_crossentropy', metrics=['accuracy'])
    return model

# Create and train the model
model = build_model(hidden_layers=[256, 128, 64], use_bias=True)
model.fit(X_train, y_train, epochs=20, batch_size=128)
